stdlib/io/src/main/java/org/ballerinalang/stdlib/io/nativeimpl/ByteChannelUtils.java

```diff
@@ -136,6 +147,56 @@ public static Object openReadableFile(String pathUrl) {
         return channel;
     }
 
+    public static Object getResource(String pathUrl) {
```

Comment:
Shall we include java doc comments for public methods?

Reply:
Java doc comments added to public methods.